Tribler/Test/Community/channel/test_channel_community.py

```diff
@@ -0,0 +1,14 @@
+from Tribler.Test.Community.channel.test_channel_base import AbstractTestChannelCommunity
+from Tribler.dispersy.util import call_on_reactor_thread
+
+
+class TestChannelCommunity(AbstractTestChannelCommunity):
+
+    @call_on_reactor_thread
```

Comment:
this is scheduling the test to run on the reactor thread and then quitting instead of waiting for the test to pass or not.